database/firebase.tsx

```diff
@@ -10,12 +10,22 @@ import { getFirestore } from "firebase/firestore";
 
 
 const firebaseConfig = {
-  apiKey: process.env.NEXT_PUBLIC_FIREBASE_API_KEY,
-  authDomain: process.env.NEXT_PUBLIC_FIREBASE_AUTH_DOMAIN,
-  projectId: process.env.NEXT_PUBLIC_FIREBASE_PROJECT_ID,
-  storageBucket: process.env.NEXT_PUBLIC_FIREBASE_STORAGE_BUCKET,
-  messagingSenderId: process.env.NEXT_PUBLIC_FIREBASE_MESSAGING_SENDER_ID,
-  appId: process.env.NEXT_PUBLIC_FIREBASE_APP_ID,
+ apiKey: "AIzaSyAyeYkaPZvmMcvzaTEKKeGMup0on3msj3Y",
+
+  authDomain: "gyan-934ec.firebaseapp.com",
+
+  databaseURL: "https://gyan-934ec-default-rtdb.firebaseio.com",
+
+  projectId: "gyan-934ec",
+
+  storageBucket: "gyan-934ec.appspot.com",
+
+  messagingSenderId: "478457042257",
+
+  appId: "1:478457042257:web:4e9bf45535974f85ba6f70",
+
+  measurementId: "G-V0LQESVYPZ"
```

Comment:
We should not expose the API keys and all these things like this. It was fine earlier. Just use the `.env` file and put these values in the variables.